core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -225,6 +128,21 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
                 stroke-linecap="round"
                 stroke-width="12"
                 ng-attr-d="<[pathSvgParameters]>"/>
+          <image ng-if="!collectionPlaythrough.hasStartedCollection()" 
+                 x="175"
+                 y="100"
+                 height="100"
+                 width="100"
+                 ng-attr-xlink:href="<[getStaticImageUrl('/general/collection_start_here_arrow.svg')]>" 
+                 xlink:href=""></image>
+          <text ng-if="!collectionPlaythrough.hasStartedCollection()" 
+                x="200"
+                y="155"
+                alignment-baseline="middle"
+                font-family="Capriola, Roboto, Arial, sans-serif"
+                font-size="20"
+                fill="#E14738"
+                text-anchor="middle">Start Here!</text>
```

Comment:
Ah, you can find the list of languages in the assets/i18n folder :) I think for now just filling in en.json, hi.json and qqq.json should be enough. We use translatewiki for the rest of our translations.

This might be useful btw: https://github.com/oppia/oppia/wiki/How-to-develop-for-i18n